source/extensions/filters/http/grpc_web/grpc_web_filter.cc

```diff
@@ -7,16 +7,54 @@
 #include "common/common/assert.h"
 #include "common/common/base64.h"
 #include "common/common/empty_string.h"
+#include "common/common/enum_to_int.h"
 #include "common/common/utility.h"
+#include "common/grpc/common.h"
 #include "common/grpc/context_impl.h"
 #include "common/http/headers.h"
 #include "common/http/utility.h"
+#include "common/runtime/runtime_features.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace GrpcWeb {
 
+namespace {
+
+// This is arbitrarily chosen.
+// TODO(dio): Make it configurable when it is needed.
```

Comment:
optional, but I don't think you need a TODO.  You can just say it's arbitrary and a config knob can be added should it be needed.

Reply:
Updated: https://github.com/envoyproxy/envoy/pull/14516/commits/041096cd830dec6101a58dc61e09c4b69cdc39ee.